README.md

```diff
@@ -37,7 +42,7 @@ There are two projects in this sample.  Each needs to be separately registered i
 3. Click on **More Services** in the left hand nav, and choose **Azure Active Directory**.
 4. Click on **App registrations** and choose **Add**.
 5. Enter a friendly name for the application, for example 'TodoListService' and select 'Web Application and/or Web API' as the Application Type. For the sign-on URL, enter the base URL for the sample, which is by default `https://localhost:44321`. Click on **Create** to create the application.
-6. While still in the Azure portal, choose your application, click on **Settings** and choose **Properties**.
+6. While still in the Azure portal, choose your application (be sure that the combo-box shows **All apps**), click on **Settings** and choose **Properties**.
```

Comment:
I've noticed that in several cases the combo-box needs to be changed to "All apps". @kalyankrishna1 did you noticed this as well?

Reply:
yes, I believe its something I noted in my excel sheet as well earlier